s
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:45:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-08-06 23:49:49 error: HTTP Exception: Validation Exception
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:45:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-08-07 07:53:09 error: JsonWebTokenError: jwt malformed
    at module.exports [as verify] (/home/vyshnav/Desktop/backend/node_modules/jsonwebtoken/verify.js:70:17)
    at authenticateMiddleware (/home/vyshnav/Desktop/backend/dist/middleware/authenticate.middleware.js:10:48)
    at Layer.handle [as handle_request] (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/layer.js:95:5)
    at /home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:284:15
    at param (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:365:14)
    at param (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:376:14)
    at Function.process_params (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:421:3)
2023-08-07 09:57:07 error: ReferenceError: crypto is not defined
    at loggerMiddleware (/home/vyshnav/Desktop/backend/dist/middleware/logger.middleware.js:8:16)
    at Layer.handle [as handle_request] (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:328:13)
    at /home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:286:9
    at Function.process_params (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:346:12)
    at next (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:280:10)
    at jsonParser (/home/vyshnav/Desktop/backend/node_modules/body-parser/lib/types/json.js:110:7)
    at Layer.handle [as handle_request] (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:328:13)
    at /home/vyshnav/Desktop/backend/node_modules/express/lib/router/index.js:286:9
2023-08-07 10:02:15 error: HTTP Exception: Validation Exception
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:39:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-08-07 10:04:20 error: HTTP Exception: Validation Exception
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:39:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-08-07 10:07:28 error: HTTP Exception: Validation Exception
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:39:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2023-08-07 11:06:05 error: HTTP Exception: Validation Exception
    at EmployeeController.<anonymous> (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:39:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/vyshnav/Desktop/backend/dist/controller/employee.controller.js:5:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
